docs/book/src/developer/providers/v1.1-to-v1.2.md

```diff
@@ -31,4 +31,9 @@ in ClusterAPI are kept in sync with the versions used by `sigs.k8s.io/controller
 
 ### Other
 
--
+- Following E2E framework functions are now checking for machine being created in the expected failure domain (if defined);
+  as a consequence, all the E2E test can now exercise failure domain too.
```

Comment:
```suggestion
  as a consequence, all E2E tests can now verify failure domains too.
```
nit. Not sure if that verb is better, exercise just sounded strange to me.

Feel free to ignore